src/main/java/com/telefonica/iot/cygnus/sinks/OrionMySQLSink.java

```diff
@@ -461,24 +480,24 @@ private MySQLAggregator getAggregator(boolean rowAttrPersistence) {
     } // getAggregator
     
     private void persistAggregation(MySQLAggregator aggregator) throws Exception {
-        String typedFieldNames = aggregator.getTypedFieldNames();
-        String fieldNames = aggregator.getFieldNames();
-        String fieldValues = aggregator.getAggregation();
+        String fieldsForCreate = aggregator.getFieldsForCreate();
+        String fieldsForInsert = aggregator.getFieldsForInsert();
+        String valuesForInsert = aggregator.getValuesForInsert();
         String dbName = aggregator.getDbName(enableLowercase);
         String tableName = aggregator.getTableName(enableLowercase);
         
         LOGGER.info("[" + this.getName() + "] Persisting data at OrionMySQLSink. Database ("
-                + dbName + "), Table (" + tableName + "), Fields (" + fieldNames + "), Values ("
-                + fieldValues + ")");
+                + dbName + "), Table (" + tableName + "), Fields (" + fieldsForInsert + "), Values ("
+                + valuesForInsert + ")");
         
         // creating the database and the table has only sense if working in row mode, in column node
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/f43ad5018cb279a5da36a91bdd130afdd99590cc